api/v1/mapping/src/commonMain/kotlin/Mappings.kt

```diff
@@ -82,6 +82,7 @@ import org.eclipse.apoapsis.ortserver.api.v1.model.SortProperty as ApiSortProper
 import org.eclipse.apoapsis.ortserver.api.v1.model.SourceCodeOrigin as ApiSourceCodeOrigin
 import org.eclipse.apoapsis.ortserver.api.v1.model.SubmoduleFetchStrategy as ApiSubmoduleFetchStrategy
 import org.eclipse.apoapsis.ortserver.api.v1.model.User as ApiUser
+import org.eclipse.apoapsis.ortserver.api.v1.model.UserDisplayName as ApiUserDisplayName
```

Comment:
Nit: In the commit message title, "names" should be singular "name". And "triggers" probably should be "triggered".